package.json

```diff
@@ -18,33 +18,31 @@
   "author": "Gregor Martynus (https://github.com/gr2m)",
   "license": "MIT",
   "dependencies": {
-    "@octokit/auth-app": "^4.0.13",
-    "@octokit/auth-unauthenticated": "^3.0.0",
-    "@octokit/core": "^4.0.0",
-    "@octokit/oauth-app": "^4.0.7",
-    "@octokit/plugin-paginate-rest": "^6.0.0",
-    "@octokit/types": "^9.0.0",
-    "@octokit/webhooks": "^10.0.0"
+    "@octokit/auth-app": "^5.0.4",
+    "@octokit/auth-unauthenticated": "^4.0.1",
+    "@octokit/core": "^5.0.0-beta.3",
+    "@octokit/oauth-app": "^5.0.0-beta.8",
```

Comment:
just fyi hold off updating to `^5.0.0` until we figure out https://github.com/octokit/core.js/pull/563#issuecomment-1599494689